chromadb/api/types.py

```diff
@@ -197,8 +198,8 @@ def __call__(self: EmbeddingFunction[D], input: D) -> Embeddings:
 
         setattr(cls, "__call__", __call__)
 
-    def embed_with_retries(self, input: D, **retry_kwargs: Dict) -> Embeddings:
-        return retry(**retry_kwargs)(self.__call__)(input)
+    def embed_with_retries(self, input: D, **retry_kwargs: Any) -> Embeddings:
```

Comment:
@suganthsolamanraja did you try to use `Dict[str,Any]` instead of Any?

Reply:
@tazarov Yes, before I tried `Dict[str, Any]`. Should I make changes?

Reply:
> @tazarov Yes, before I tried `Dict[str, Any]`. Should I make changes?

previously, you said. "would it make sense to leave the return type as Embeddings and for the kwargs use Any"